packages/block-editor/src/components/global-styles/use-global-styles-output.js

```diff
@@ -1106,8 +1106,11 @@ const processCSSNesting = ( css, blockSelector ) => {
 	return processedCSS;
 };
 
-export function useGlobalStylesOutput() {
+export function useGlobalStylesOutput( customMergedConfig = null ) {
 	let { merged: mergedConfig } = useContext( GlobalStylesContext );
+	if ( !! customMergedConfig ) {
```

Comment:
The `!!` is unnecessary in an `if`.

```suggestion
	if ( customMergedConfig ) {
```